src/rust/engine/fs/src/store.rs

```diff
@@ -1019,6 +1052,48 @@ mod local {
     pub fn all_lmdbs(&self) -> Vec<(Arc<Environment>, Database, Database)> {
       self.lmdbs.values().cloned().collect()
     }
+
+    pub fn compact(&self) -> Result<(), String> {
+      for b in 0x00..0x10 {
```

Comment:
Should this range be constant? Or the bounds at least?

Reply:
Pulled out fn